app/controllers/application_controller/ci_processing.rb

```diff
@@ -1049,6 +1051,8 @@ def process_vm_buttons(pfx)
     when "#{pfx}_live_migrate"              then livemigratevms
     when "#{pfx}_associate_floating_ip"     then associate_floating_ip_vms
     when "#{pfx}_disassociate_floating_ip"  then disassociate_floating_ip_vms
+    when "#{pfx}_add_security_group"        then add_security_group_vms
+    when "#{pfx}_remove_security_group"     then remove_security_group_vms
```

Comment:
Is this going to be relevant for multiple different entities?

What I mean: you are adding some "#{pfx}_action" here. It seems to be relevant for VMs, right? Is it going to be relevant for other entities?

Or is it just that you saw many of the VM button action being handled here?

If it's going to be relevant just for VMs then I'd prefen not to do the `pfx` magic.

But if you actually rewrite this to be pure client-side throught the API then you probably will not need to change this file (and the one below) at all.

Reply:
@martinpovolny hi! to be honest, I don't know what this code is for; would you suggest just removing this bit?